learning_journal/views.py

```diff
@@ -35,9 +36,16 @@ def create(request):
         return HTTPFound(location=request.route_url('home'))
     return {'form': form, 'action': request.matchdict.get('action')}
 
-@view_config(route_name='action', match_param='action=edit', renderer='string')
+@view_config(route_name='action', match_param='action=edit', renderer='templates/edit.jinja2')
 def update(request):
-    return 'edit page'
+    string = request.query_string
+    __, req_id = string.split('=')
```

Comment:
There's an easier way to handle this.  Check out the attributes of the Pyramid `request` object: http://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html, and in particular, the [`params`](http://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.params) attribute.